docs/how-to-guides/running-feast-in-production.md

```diff
@@ -77,16 +116,38 @@ In the above example we are materializing the source data from the `driver_hourl
 
 The timestamps above should match the interval of data that has been computed by the data transformation system.
 
-## 3. How to use Feast for model training and serving
+### 2.2. Automate periodic materializations 
+
+It is up to you which orchestration/scheduler to use to periodically run `$ feast materialize`.
+Feast keeps the history of materialization in its registry, so the choice could be as simple as a [unix cron util](https://en.wikipedia.org/wiki/Cron).
+However, with this option materialization work is not distributed, and you might end up limited by the resources of a single machine. 
```

Comment:
Should we talk about HOW materialization works and why it's a problem to use Cron? Not clear from these docs.